sdk/extensions/Microsoft.Extensions.Azure/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.6.0-beta.1 (Unreleased)
+## 1.6.0 (2022-09-22)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Support passing `additionallyAllowedTenants` via config.
```

Comment:
Should we mention that this could potentially break some stuff and describe how/what?   Maybe link back to the Identity write-up for things?

Reply:
Technically it's already broken, but we could just link to the Identity breaking change doc here. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/BREAKING_CHANGES.md#170